cognite/neat/workflows/_exceptions.py

```diff
@@ -0,0 +1,8 @@
+from cognite.neat.utils.exceptions import NeatError
+
+
+class InvalidStepOutputException(NeatError):
```

Comment:
@alivinco are you raising any exception message here ?

Reply:
The message is set with dynamic details  when rising exception